Makefile

```
@@ -119,15 +119,15 @@ endif
 dracut.8: man/dracut.8.asc \
 	man/dracut.usage.asc
 
-dracut.html: man/dracut.asc $(manpages) dracut.css man/dracut.usage.asc
+dracut.html: man/dracut.asc $(manpages) docs/dracut.css man/dracut.usage.asc
```

of you generate with ` -o docs/dracut.html` then this becomes:
```suggestion
docs/dracut.html: man/dracut.asc $(manpages) docs/dracut.css man/dracut.usage.asc
```

and the other `dracut.html` dependencies have to be changed as well